cmd/release/build.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func build(date, tag string) ([]string, error) {
+	outputs := []string{}
+	for goos, goarchs := range targets {
+		for _, goarch := range goarchs {
+			log.Println(fmt.Sprintf("==> Build on %s for %s ...", goos, goarch))
+			target := fmt.Sprintf("dist/terraform-provider-ansiblevault_%s-%s_%s", goos, goarch, tag)
```

Comment:
`dist/` is hardcoded here and might be hard to find. Can you make a constant or a flag with default value for this then use `path.Join` for concatenate those paths ?

Reply:
Yes I can

Reply:
Done c5d3151